spec/openapi.yaml

```diff
@@ -339,113 +215,40 @@ paths:
                   users:
                     type: array
                     items:
-                      $ref: "#/components/schemas/OctoUsers"
-        403:
-          $ref: "#/components/responses/Unauthorized"
-  /api/downloads/{target}/{filename}:
```

Comment:
This is compatible with Octoprint. Why should remove it?

Reply:
Readded